kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/TestRefinableViews.java

```diff
@@ -17,19 +17,25 @@
 package org.kitesdk.data.spi;
 
 import com.google.common.io.Closeables;
+
 import java.io.IOException;
 import java.util.concurrent.Callable;
+
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 import org.kitesdk.data.*;
 import org.kitesdk.data.event.StandardEvent;
+
 import com.google.common.collect.Sets;
+
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Set;
+
```

Comment:
Oops, next time around I'm definitely going to invest some more time making sure eclipse isn't quite so opinionated.